ipmitool/ipmitool.spec

```diff
@@ -1,14 +1,14 @@
 Name:         ipmitool-xcat
 Summary:      ipmitool - Utility for IPMI control
-Version:      1.8.15
-Release:      3
+Version:      1.8.17
+Release:      0
 License:      BSD
 Group:        Utilities
 Packager:     IBM Corp.
 Source:       ipmitool-%{version}.tar.gz
 Patch:        ipmitool-%{version}-saneretry.patch
 Patch2:       ipmitool-%{version}-rflash.patch
-Patch3:       ipmitool-%{version}-solactivate.patch
+#Patch3:       ipmitool-%{version}-solactivate.patch
```

Comment:
I'm sure that the code in the solactivate.patch is not included in the 1.8.17 version.